src/components/Person/Person.jsx

```diff
@@ -1 +1,24 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+      {age && `<p className="Person__age"> I am ${age}</p>`}
+      <p className="Person__partner">
+        {isMarried ? (`${partnerName} is my ${sex === 'm' ? 'wife' : 'husband'}`)
```

Comment:
Pay attention to these two requirements. Create a variable for wife/husband
![image](https://user-images.githubusercontent.com/96409563/225398818-51afde10-495d-41ca-8f30-36f260d7ce0f.png)